src/Main.java

```diff
@@ -8,22 +8,21 @@ public static void main(String[] args) {
             System.out.println("Установите версию приложения для Android по ссылке");
         }
         System.out.println("Задание 2");
-        int clientDeviceOS1 = 0;
-        int clientDeviceYear = 2014;
-        if ((clientDeviceOS1 == 0) && (clientDeviceYear < 2015)) {
-                System.out.println("Установите облегченную версию приложения для iOS по ссылке");
-            } else {
-                System.out.println("Установите версию приложения для iOS по ссылке");
-            }
-        if ((clientDeviceOS1 == 1) && (clientDeviceYear < 2015)) {
-                System.out.println("Установите облегченную версию приложения для Android по ссылке");
-            } else {
-                System.out.println("Установите версию приложения для Android по ссылке");
-            }
+        int clientOS1 = 0;
```

Comment:
зачем опять новая переменная-то? Мы же уже обсудили это, и опять осталось всё. Вот так надо было сделать
```suggestion
        clientOS = 0;
```
и всё. Просто присвоить другое значение уже СОЗДАННОЙ ранее переменной